src/Profile/EditProfile.tsx

```diff
@@ -57,72 +58,80 @@ const CreateOrEditProfile = (props: Props) => {
         let passwordLengthError: boolean = false;
         let passwordMatchError: boolean = false;
         let phoneNumberError: boolean = false;
-
+        let alphanumericError: boolean = false;
 
         if (!profile.firstName || profile.firstName === "") {
             nameError = true;
         }
         if (!profile.lastName || profile.lastName === "") {
             lastNameError = true;
         }
-        if (!password || password.length<7) {
+        if (!profile.password || profile.password.length<7) {
             passwordLengthError = true;
         }
-        if (password !== confirmPassword) {
+        if (profile.password !== confirmPassword) {
             passwordMatchError = true;
         }
         if (!profile.phoneNumber || profile.phoneNumber === "") {
             phoneNumberError = true;
         }
+        if (!profile.password || !regexp.test(profile.password)){
+            alphanumericError = true;
+        }
 
         const newErrors: Errors = {
             nameError,
             lastNameError,
             passwordLengthError,
             passwordMatchError,
-            phoneNumberError
+            phoneNumberError,
+            alphanumericError
         }
 
         return newErrors;
     }
 
     return (
-        <div className={"create-book"}>
+        <div >
             <div className={"update-profile-title"}>{'Mis Datos'}</div>
-            {(errors.nameError && renderError("Completar nombre")) ||
-            (errors.lastNameError && renderError("Completar apellido")) ||
-            (errors.passwordLengthError && renderError("La contraseña debe tener más de 6 caracteres!")) ||
-            (errors.passwordMatchError && renderError("Las contraseñas deben coincidir!")) ||
-            (errors.serverError && renderStatusError(errors.serverError))
-            }
             <div>
+                <div className={"box"}>
+                    {(errors.nameError && renderError("Completar nombre")) ||
+                    (errors.lastNameError && renderError("Completar apellido")) ||
+                    (errors.passwordLengthError && renderError("La contraseña debe tener más de 6 caracteres!")) ||
+                    (errors.passwordMatchError && renderError("Las contraseñas deben coincidir!")) ||
+                    (errors.alphanumericError && renderError("La contraseña solo puede incluir letras y/o números!")) ||
+                    (errors.serverError && renderStatusError(errors.serverError))
+                    }
```

Comment:
In this case, that you have some different types of error, you can create a function to render the corresponding error.